Magento_Checkout/layout/checkout_cart_index.xml

```diff
@@ -5,36 +5,33 @@
 >
     <body>
         <!-- main column -->
-        <referenceContainer name="checkout.cart.container"
-                            htmlClass="cart-container grid cart"
-        >
+        <referenceContainer name="checkout.cart.container" htmlClass="cart-container grid cart">
             <container name="cart.main.container"
                        as="cart_main_container"
                        label="Shopping Cart Main Container"
                        htmlTag="div"
                        htmlClass="grid__column grid__column--4-6 cart__main-col"
-            />
-            <block class="Magento\Checkout\Block\Cart"
-                   name="checkout.cart.form"
-                   as="cart-items"
-                   template="cart/form.phtml"
-                   before="cart.summary"
             >
-                <block class="Magento\Framework\View\Element\RendererList"
-                       name="checkout.cart.item.renderers"
-                       as="renderer.list"
-                />
-                <block class="Magento\Framework\View\Element\Text\ListText"
-                       name="checkout.cart.order.actions"
+                <container name="cart.main.gift.coupon"
+                           as="cart_main_gift_coupon"
+                           label="Shopping Cart Main Container For Fift Card And Coupon"
```

Comment:
Typo in label. I think it should be Gift not Fift.

Reply:
fixed